robottelo/constants/__init__.py

```diff
@@ -676,8 +676,8 @@
     },
     'Include Last Reboot': {'yes': 'yes', 'no': 'no'},
 }
-DOCKER_REGISTRY_HUB = 'https://registry-1.docker.io'
-DOCKER_UPSTREAM_NAME = 'busybox'
+DOCKER_REGISTRY_HUB = 'https://quay.io'
+DOCKER_UPSTREAM_NAME = 'projectquay/quay'
```

Comment:
```suggestion
DOCKER_UPSTREAM_NAME = 'quay/busybox'
```
Can you test with that and compare the time?

Reply:
Updated to use `quay/busybox`.

```
$ pytest -k TestDockerRepository tests/foreman/api/test_repository.py
[...]
collected 229 items / 212 deselected / 17 selected

tests/foreman/api/test_repository.py .................                                                                                                                                                                                 [100%]
[...]
=== 17 passed, 212 deselected, 59 warnings in 107.81s (0:01:47) ===
```